lib/core/game.rb

```diff
@@ -0,0 +1,49 @@
+class Game
+  def initialize(options)
+    @board = options[:board]
+    @players = options[:players]
+    @ui = options[:ui]
+  end
+
+  def play
+    ui.greet_user
+    play_round
+  end
+
+  private
+  attr_reader :players
+  attr_reader :ui
+  attr_accessor :board
+
+  def play_round
+    while !board.is_terminal? do
```

Comment:
The method name `is_terminal?` may be confusing, I thought you were referring to the console.

Reply:
I'd never thought of that, good point!